dev/bots/analyze.dart

```diff
@@ -1021,7 +1015,7 @@ Future<void> verifyNoBinaries(String workingDirectory, { Set<Hash256> legacyBina
   assert(
     _legacyBinaries
       .expand<int>((Hash256 hash) => <int>[hash.a, hash.b, hash.c, hash.d])
-      .reduce((int value, int element) => value ^ element) == 0x606B51C908B40BFA // Please do not modify this line.
+      .reduce((int value, int element) => value ^ element) == 0xA88D8E5183D353F9 // Please do not modify this line.
```

Comment:
@Hixie The spirit of this "do not modify" seems to be to avoid adding more binaries, not removing.

Reply:
We should leave the hashes. That way if we ever add those binaries back it's fine.
There's no value to removing the hashes because removing the files doesn't actually remove the files.

Reply:
I wasn't trying to remove the hashes, I'm just assuming I can violate the "Please do not modify this line." to update the hash if it's to remove legacy binaries.

Reply:
Oh I see, you want me to leave them even if the integration test was removed?

Reply:
Reverted changes to the hashes.

Reply:
> Oh I see, you want me to leave them even if the integration test was removed?

yeah. sorry i wasn't clear.